### ISSUE TITLE
Review existing ban on "screenshot"?

### ISSUE THREAD

**Issue by julian-cable** [StyleGuides/WritingStyleGuide#540]
The guide currently bans "screenshot" in favour of "screen capture". That guidance reflects what IBM Style specified many years ago, but in recent years, IBM has allowed either term. Should we now follow suit and also allow either term?

**Comment by sffrench** [comment]
I choose screenshot, I feel it is the most common name for it, but also don't mind the use of both together.

**Comment by julian-cable** [comment]
I suggest allowing either term. I would hesitate to ban a term in the absence of any concerns about its usage.

**Comment by rclee33** [comment]
I'm fine with allowing either term. I'm also fine with preferring one over the other. Personally, what I think is most important is removing the references to the IBM Style guide (for these word usage entries) if we deviate from their usage. 

There's an interesting discussion here: https://english.stackexchange.com/questions/280448/screenshot-vs-screen-capture

TLDR: One possible difference between the two terms is that "screencapture" refers to the action of capturing the screen (either still image or video), while "screenshot" refers to the outcome of the action (the image itself). To be clear, I don't think we necessarily need to abide by this or include any of that in our own guidance. I just thought it was interesting.

**Comment by julian-cable** [comment]
@rclee33. For this entry we need to remove "See the relevant entry in the IBM Style Guide." 
Thanks also for sharing that interesting link. 
I think that nowadays, "screenshot" is the more commonly used of the two terms. Merriam-Webster has an entry for "screenshot" but not for "screen capture". The latter term would have been more used 10-15 years ago, but not so much now.

**Comment by julian-cable** [comment]
If we must choose one term to use, I would go for "screenshot", based on its common usage.

**Comment by julian-cable** [comment]
Change of guidance to implement for the next release.

**Comment by rclee33** [comment]
Just noting that if we update the style guide to allow "screenshot", we'll need to update aspell as well. Currently, aspell flags "screenshot" and suggests variations that are not acceptable: "screen shot" and "screen-shot". 

What's the next step here, opening an issue on the aspell repo and linking it to this one?

**Comment by PatSharp99** [comment]
It seems like you have all come to a resolution, to use just "screenshot" and update the style guide, remove the reference to the IBM style guide, and update aspell. Please execute this as you have described. Thank you.

**Comment by julian-cable** [comment]
Yes indeed, this is in scope for the 6.2 release.
